src/users/interfaces/add-user-to-organization-options.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface AddUserToOrganizationOptions {
+  id: string;
```

Comment:
Perhaps this can make it clearer the object for this ID?

```suggestion
  user_id: string;
```

Reply:
ok i started out with user_id and switched to `id` to match the api docs, but i agree it's more clear like this, will fix.

Reply:
(maybe we should update the docs too)

Reply:
Hmm the docs have the context that `:id` is part of the URL which is like 10 pixels above the definition of `id`. Having it be `:user_id` just for this endpoint would make it differ from other user endpoints that are just `:id`, and from what I can tell, all our point-load APIs have just `:id` in the URL for the docs.

I think it's ok in this case if the SDK deviates from the API docs, since there's no context within the SDK which ID is used where at the function signature level.

Reply:
ok, i've replaced all the `id`s with `user_id` for consistency.